system-tests/lib/protocolStub.js

```diff
@@ -0,0 +1,33 @@
+/* global Debug */
+
+const AppCaptureProtocol = class {
```

Comment:
Some function signatures have changed and some functions aren't being implemented. We should probably just implement the interface to make sure it stays in sync.

```suggestion
import type { AppCaptureProtocolInterface } from '@packages/types'

const AppCaptureProtocol = class implements AppCaptureProtocolInterface {
```

Reply:
This is a `js` file, I think it was just intended as a fake for the signing test stuff

Reply:
Updated to make sure we're in line if need be: [`74c5ffa` (#26421)](https://github.com/cypress-io/cypress/pull/26421/commits/74c5ffab1a81df849130510e0f84c23c9250ef5d)